README.md

```diff
@@ -889,6 +889,38 @@ targets:
   - name: gem
 ```
 
+### AWS Lambda Layer
+
+The target will create a new public lambda layer in each available region with
+the extracted artifact from GitHub actions.
+
+**Environment**
+| Name | Description |
+| --------------------- | -------------------------------------------------------------------------- |
+| AWS_ACCESS_KEY | The access key of the AWS account to create and publish the layers. |
+| AWS_SECRET_ACCESS_KEY | The secret access key of the AWS account to create and publish the layers. |
+
+**Configuration**
+| Option | Description |
+| -------------- | ------------------------------------------------------------------------ |
+| includeNames | **optional** Pattern (between `/`) of the ZIP artifact to be used as the content of the layer. If after filtering the artifacts with this option there's no exactly one artifact, it'll fail. |
```

Comment:
`includeNames` exists for all targets, so perhaps we can link to https://github.com/getsentry/craft#per-target-options and mention the default value here?